src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -92,6 +92,19 @@ interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExec
     """,
   )
   fun getLicencesPassedExpiryDate(): List<Licence>
+
+  @Query(
+    """
+    SELECT l
+        FROM Licence l
+        WHERE l.statusCode  IN (
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.IN_PROGRESS,
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.SUBMITTED
+        )
+        AND COALESCE(l.conditionalReleaseDate, l.actualReleaseDate) < CURRENT_DATE
```

Comment:
I think the params for `COALESCE` should be reversed. ie:
`COALESCE( l.actualReleaseDate, l.conditionalReleaseDate) < CURRENT_DATE`
as we prioritise actualReleaseDate when determining the release date, and it's more likely to be null.

Reply:
I wonder if we have the same issue on the time out job 👁️

Reply:
looks ok on there: 
`    WHERE COALESCE(l.actualReleaseDate, l.conditionalReleaseDate) <= :timeOutDate`

Reply:
removed coalesce as it wasn't useful when release date was not null and in future (theoretical possibility) and crd was in past